src/pages/references/node-configuration.md

```diff
@@ -0,0 +1,489 @@
+---
+title: Node Configuration
+description: Configuration parameters and options for the stacks-node binary
+icon: TestnetIcon
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Subcommands](#subcommands)
+- [Configuration File Options](#configuration-file-options)
+  - [Section: node](#section-node)
+  - [Section: events_observer (optional)](#section-events_observer-optional)
+  - [Section: connection_options (optional)](#section-connection_options-optional)
+  - [Section: burnchain](#section-burnchain)
+  - [Section: mstx_balance](#section-mstx_balance)
+
+## Installation
+
+See this page for installation instructions.
+[@page-reference | inline]
+| /stacks-blockchain/running-testnet-node
+
+## Usage
+
+```bash
+stacks-node sub-command [--subcommand-option <value>]
+```
+
+## Subcommands
+
+### mocknet
+
+Start a node based on a fast local setup emulating a burnchain. Ideal for smart contract development.
+
+Example:
+
+```bash
+stacks-node mocknet
+```
+
+### krypton (deprecated)
+
+Start a node that will join and stream blocks from the public krypton testnet, powered by Blockstack via (Proof of Transfer).
+
+Example:
+
+```bash
+stacks-node krypton
+```
+
+### xenon
+
+Start a node that will join and stream blocks from the public xenon testnet, decentralized.
+
+Example:
+
+```bash
+stacks-node xenon
+```
+
+### start
+
+Start a node with a config of your own. Can be used for joining a network, starting new chain, etc.
```

Comment:
Can be used for joining a network, starting new chain, replacing default values for `mocknet` or `xenon`.